qiskit_nature/algorithms/eigen_solvers/q_eom_ee.py

```diff
@@ -27,7 +27,7 @@
 class QEomEE(NumPyMinimumEigensolver):
```

Comment:
I am actually surprised to see this class as well as `QEomVQE` and `QEquationOfMotion`. All of these classes should have been deprecated when we released `qiskit.chemistry.algorithms.excited_state_solvers.QEOM`.

@paulineollitrault Please verify the above

If this is correct, @manoelmarques please remove the entire `qiskit_nature.algorithms.eigen_solvers` module :+1:

Reply:
The lack of deprecation was most likely just an oversight given all the work that went on. It was my understanding that the excited state algorithms were being used now and were the only ones that were wanted.